openvidu-browser/src/OpenVidu/OpenVidu.ts

```diff
@@ -43,7 +43,11 @@ import RpcBuilder = require('../OpenViduInternal/KurentoUtils/kurento-jsonrpc');
  */
 import platform = require('platform');
 
-platform['isIonicIos'] = (platform.product === 'iPhone' || platform.product === 'iPad') && platform.ua!!.indexOf('Safari') === -1;
+const isTouchable = 'ontouchend' in document;
```

Comment:
Concerned about having this code at init of the file. Are you sure platform will be available? Maybe better to keep it inside the function as it was?

Reply:
Yes from my debugging sessions I could see that platform was initialised, but as mentioned, `platform.product` isn't set to `iPad` for iPad Pro's. The check for iPad in `platform['isIonicIos']` was never in a function before. We've had "our own" function to check for iPad's in other places though.